docs/content/users/quickstart.md

```diff
@@ -222,10 +222,11 @@ DDEV comes ready to work with any PHP project, and has deeper support for severa
         ```bash
         mkdir my-typo3-site
         cd my-typo3-site
-        ddev config --project-type=typo3 --docroot=public --create-docroot
+        ddev config --project-type=typo3 --docroot=public --create-docroot --php-version 8.1
         ddev start
         ddev composer create "typo3/cms-base-distribution" --no-install
         ddev composer install
+        ddev config --auto
```

Comment:
```suggestion
        ddev restart
```

When you commit these, just add [skip ci] in the commit message. You can commit the two together as a batch.